src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,123 @@
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
+  return (
+    filteredTodos.length === 0
+      ? (
+        <p className="notification is-warning">
+          There are no todos matching current filter criteria
+        </p>
+      )
+      : (
+        <table className="table is-narrow is-fullwidth">
+          <thead>
+            <tr>
+              <th>#</th>
+              <th>
                 <span className="icon">
-                  <i className="far fa-eye" />
+                  <i className="fas fa-check" />
                 </span>
-              </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </>
+              </th>
+              <th>Title</th>
+              <th> </th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {filteredTodos.map((todo: Todo) => (
+              <tr
+                data-cy="todo"
+                key={todo.id}
+                className={classNames(
+                  {
+                    'has-background-info-light':
+                      openedTodo && openedTodo.id === todo.id,
+                  },
+                )}
+              >
+                <>
+                  <td className="is-vcentered">{todo.id}</td>
+                  <td className="is-vcentered">
+                    {
+                      todo.completed && (
+                        <span className="icon" data-cy="iconCompleted">
+                          <i className="fas fa-check" />
+                        </span>
+                      )
+                    }
```

Comment:
it's not an issue just a recommendation for better readability
```suggestion
                    {todo.completed && (
                      <span className="icon" data-cy="iconCompleted">
                        <i className="fas fa-check" />
                      </span>
                    )}
```